src/ValueObject/AutoEncryptionOptions.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace MongoDB\ValueObject;
```

Comment:
This class should move to the `MongoDB\Model` namespace, which contains similar classes.